server/src/controllers/Events/resolver.ts

```diff
@@ -823,17 +798,32 @@ ${unsubscribeOptions}`,
     });
     await deleteEventReminders(id);
 
+    const chapterURL = `${process.env.CLIENT_LOCATION}/chapters/${event.chapter.id}`;
+    const eventURL = `${process.env.CLIENT_LOCATION}/events/${event.id}`;
     const notCanceledRsvps = event.event_users;
-
+    const unsubScribeOptions = NotificationContextText({
+      linkToEvent: eventURL,
+      linkToChapter: chapterURL,
+    });
```

Comment:
This is complicated because this cancel RSVP without userId, so if we are using `getEventUnsubscibeOptions`, we will force user to log in

Reply:
Should we move the text to the resolver with comment of why we are doing this?

Reply:
We can't force users to login when they want to unsubscribe from email notifications.  Fortunately, there's no need to.  We have access to the userId via `event_users`.  Since each email is different (different unsubscribe tokens) you can use `batchSender` to send them all.  There's an example of that in `sendEventInvite`.